cg/io/config.py

```diff
@@ -0,0 +1,17 @@
+"""Module to read or write config files"""
+
+from pathlib import Path
+from typing import Any
+from cg.constants.symbols import EMPTY_STRING
+
+
+def write_config_nextflow_style(content: dict[str, Any] | None) -> str:
+    """Write content to stream accepted by Nextflow config files with non-quoted booleans and quoted strings."""
+    string = EMPTY_STRING
+    double_quotes = '"'
+    for key, value in content.items():
```

Comment:
```suggestion
    for key, file_path in content.items():
```
perhaps a better descritpor for key also cannot make out what that would be

Reply:
Actually the function should be completely independent of the value: can be a boolean flag parameter, a path, a string etc. So I would keep value. But key can be changed to "parameter"